### PR TITLE
Mark video APIs as beta

### PR DESCRIPTION
**Pull request by shilcare**
Mark video APIs as beta. 
Resolves MicrosoftDocs/msteams-docs#6121

### CODE REVIEW

**Review by ydogandjiev — DISMISSED** [review]

**Review by manasisharma — DISMISSED** [review]

**Comment by MarcinJuraszek** [comment]
This API shipped as part of v1, in _public_ folkder, without a @beta tag. So I'm not sure I understand how it's not really 'public'. It is. It shipped. Making it @beta or hiding it is a breaking change, as customers might have had taken a dependency on it. What am I missing here?

cc: @erinha

**Comment by shilcare** [comment]
> This API shipped as part of v1, in _public_ folder, without a @beta tag. So I'm not sure I understand how it's not really 'public'. It is. It shipped. Making it @beta or hiding it is a breaking change, as customers might have had taken a dependency on it. What am I missing here?
> 
> cc: @erinha
Video APIs should not have been published as they are still in internal rings and are gated in Teams, so only a few known partners have access to these APIs for now.

**Comment by tdwhite0** [comment]
@MarcinJuraszek Yes this has been public in the code for a long time, it has never actually worked in Teams though or been officially documented. I've been watching this feature for a long time and have built some things with this: https://github.com/microsoft/teams-videoapp-sample. I've been hoping it would work eventually, so seeing it now marked hidden is a bummer.

**Comment by erinha** [comment]
> > This API shipped as part of v1, in _public_ folder, without a @beta tag. So I'm not sure I understand how it's not really 'public'. It is. It shipped. Making it @beta or hiding it is a breaking change, as customers might have had taken a dependency on it. What am I missing here?
> > cc: @erinha
> 
> Video APIs should not have been published as they are still in internal rings and are gated in Teams, so only a few known partners have access to these APIs for now.

As per the comments on this PR #1213, unfortunately we've already shipped these APIs, so I think we're committed to supporting them, and hiding the documentation doesn't achieve this. I suggest you attend office hours to discuss your feature and its roadmap, given that it has already been publicly released and has not been previously reviewed. https://aka.ms/extendingteamsjssdk

**Review by ydogandjiev — DISMISSED** [review]

**Review by erinha — CHANGES_REQUESTED** [review]
Comments about applying the beta tag to all functions/interfaces/enums/etc. within the `video` namespace.

File: change/@microsoft-teams-js-b58082cb-6568-4c24-89f6-c55c4ec296ff.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "mark video api as beta",
```
[the remark this answers is not in the document]
Comment (shilcare) [reply change/@microsoft-teams-js-b58082cb-6568-4c24-89f6-c55c4ec296ff.json]: updated

**Review by erinha — APPROVED** [review]

**Review by ydogandjiev — APPROVED** [review]

**Review by erinha — APPROVED** [review]

**Pull request merged**